static/styles/compose.css

```diff
@@ -798,6 +783,44 @@ a.compose_control_button.hide {
     margin: auto;
 }
 
+#compose_stream_selection_dropdown {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+
+    .stream_header_colorblock {
+        margin: 0;
+    }
+
+    .dropdown-toggle {
+        border-radius: 1px 4px 4px 1px !important;
```

Comment:
The other `.dropdown-toggle` in the codebase also have this `!important`, but it seems like it might not be necessary. I could look into that separately from this change.